cpp/src/arrow/compute/exec/CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ add_arrow_compute_test(util_test
 
 add_arrow_benchmark(expression_benchmark PREFIX "arrow-compute")
 
+add_arrow_benchmark(project_benchmark PREFIX "arrow-compute")
```

Comment:
Minor nit: Since we are benchmarking nodes and not the kernel functions we might want to start using a new prefix just to help distinguish.  Maybe `arrow-engine`?

CC @lidavidm for second opinion.

Reply:
Practically everything is still under `arrow::compute` so I don't think it's a big deal. If/when we ever refactor the code structure we can fix it then.